### ISSUE TITLE
Allow import of big gitlab projects

### ISSUE THREAD

**Issue by Gredin67** [YunoHost-Apps/gitlab_ynh#185]
### Describe the bug

trying to import a gitlab project, I get 413 Request Entity Too Large although the file weights 477M and the limit in /etc/gitlab/gitlab-persistent.rb is nginx['client_max_body_size'] = "1000m"
I also tried adding client_max_body_size 500m; in /etc/nginx/nginx.conf

**Comment by kay0u** [comment]
After changing the file `gitlab-persistent.rb` did you launch the command:
```
sudo gitlac-ctl reconfigure
```
And after changing the file `/etc/nginx/conf.d/you.gitlab.domain.d/gitlab.conf`, did you reload the nginx config with:
```
sudo systemctl reload nginx
```

**Comment by farvardin** [comment]
hello @kay0u 

thanks a lot for the hint about /etc/nginx/conf.d/you.gitlab.domain.d/gitlab.conf

it seems to do the trick for me... 

First I was modifying 
``/etc/yunohost/apps/gitlab/conf/nginx.conf`` instead (it's the only relevant file I got from a search for nginx.conf)

What are the difference between the 2 files? They look similar, except for the part I've altered and a thing about proxy_pass. 

What is the use of /etc/yunohost/apps/gitlab/conf/nginx.conf if it's not parsed as expected?
